routes/api.php

```diff
@@ -27,3 +27,7 @@
     ->except("index")
     ->middleware('auth:sanctum');
 Route::get("/categories" , [\App\Http\Controllers\API\CategoryController::class , "index"])->name("categories.index");
+
+
+
+Route::put("categories/{id}",[\App\Http\Controllers\API\CategoryController::class , "update"])->name("categories.update");
```

Comment:
You don't need this
If you look at the top line, we have this
```
Route::apiResource("/categories" , \App\Http\Controllers\API\CategoryController::class)
    ->except("index")
    ->middleware('auth:sanctum');
```
There are also update items in `apiResource`
Use `php artisan route:list` to see the list of routes and research about `apiResource`

Reply:
ok,thanks.